tensorboard/webapp/metrics/store/metrics_store_internal_utils.ts

```diff
@@ -370,5 +370,19 @@ export function canCreateNewPins(state: MetricsState) {
     state.unresolvedImportedPinnedCards.length;
   return pinCountInURL < util.MAX_PIN_COUNT;
 }
+/**
+ * Sets cardStepIndex for image card based on selected time.
+ */
+export function generateNextCardStepIndexFromSelectTime(
+  previousCardStepIndex: CardStepIndexMap
+): CardStepIndexMap {
+  const nextCardStepIndex = {...previousCardStepIndex};
+
+  Object.keys(previousCardStepIndex).forEach((cardId) => {
```

Comment:
I think you have been a bit too aggressive in breaking this up into smaller CLs - would be nice to have at least something worth testing :). I think this does nothing. If I'm right, please delete it. If I'm wrong, please correct me and add a comment.

Having said that, if this logic does nothing and you remove it then I approve of this PR.

Reply:
I added this to make the method meaning. As stated in the jsdoc, it's meant to be image card only. 
Let me know if you still prefer to remove this (which will leave the jsdoc a bit unexplained, but if you're okay with this I'm okay too)